src/components/GsocTsoc/GsocTsoc.jsx

```diff
@@ -0,0 +1,239 @@
+import React, { useState, useEffect } from "react";
+import { Table, NavLink, Input, Badge, Row, Col, Alert } from "reactstrap";
+import Carousel from "react-multi-carousel";
+import "react-multi-carousel/lib/styles.css";
+import ProjectModal from "../ProjectModal/ProjectModal";
+import defaultcardimg from "../../../static/logos/defaultCardcover.jpg";
+
+const GsocTsoc = () => {
```

Comment:
Be consistent with variable naming conventions, we're using camel-case variables everywhere else.